segmentation/utils/segmentation_utils_test.py

```diff
@@ -62,8 +62,48 @@ def _create_test_extraction_data():
     return cell_mask, channel_data
 
 
+def test_compute_complete_expression_matrices():
```

Comment:
We can add some simple tests for just running the function with different values to make sure the various conditionals are working as intended

Reply:
forgot to mention this in my review, sorry!

Two things fix the coverage loss:
 - adding a `points=None` case for mibitiffs
 - adding more points (and/or lowering batch size) so that the 'batching-for-loop' is entered during testing